media/css/mozorg/home/home-a.less

```diff
@@ -0,0 +1,1097 @@
+@import '../../pebbles/lib.less';
+
+// Variables and Mixins
+
+@color-mozred-bright: saturate(lighten(@color-mozred, 5%), 10%);
+
+.invert-text() {
+    color: #fff;
+
+    a:link,
+    a:visited,
+    a:hover,
```

Comment:
if we make this a more global mixin we probably should, maybe at least an underline on hover and focus, but for how it's used on this page at the moment it's really just shorthand for "make this stuff white." A default hover/focus style would likely have to be overridden.